models/sales/sales.yml

```diff
@@ -143,6 +143,15 @@ models:
       - name: deal_name
         description: The name you have given this deal.
 
+      - name: is_deal_pipeline_deleted
+        description: Whether the record was deleted from the deal pipeline table.
+
+      - name: is_deal_pipeline_stage_deleted
+        description: Whether the record was deleted from the deal pipeline stage table.
+
+      - name: is_deal_deleted
```

Comment:
we also need to replace `is_deleted` and `_fivetran_deleted` with `is_<table name>_deleted` in the other models (ie contact_lists, contacts, comapnies etc)